KEYS

```diff
@@ -126,3 +126,286 @@ txTq7YpleWQhcz9+9Fruu7jA+l1pSUJSR0+DZegBOq+zWIHcZSTbAnfOX+jYySYd
 lsw/
 =GJFW
 -----END PGP PUBLIC KEY BLOCK-----
+pub   dsa1024 2007-06-30 [SC]
+      50D7C82AC19334EA3C75699AF39F187DEFB55DF1
+uid           [ unknown] Luciano Resende (Code Signing Key) <lresende@apache.org>
+sig 3        F39F187DEFB55DF1 2007-06-30  Luciano Resende (Code Signing Key) <lresende@apache.org>
+sig 2        F4268EE7F2EFD0F0 2010-11-05  Christopher David Schultz (Christopher David Schultz) <chris@christopherschultz.net>
```

Comment:
@lresende : Not sure if sig entries for other users must be present. Can  you remove them if not needed ?